coalib/testing/LocalBearTestHelper.py

```diff
@@ -185,10 +201,11 @@ def check_results(self,
                                   create_tempfile=create_tempfile,
                                   tempfile_kwargs=tempfile_kwargs,
                                   settings=settings)
+        self.addTypeEqualityFunc(list, self.assertObjectsEqual)
```

Comment:
We could actually use `Iterable` instead of only `list`, so we support any kind of iterable :+1: